client/src/tests/App.test.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { shallow, mount, render } from 'enzyme';
+import App from '../components/App.jsx';
+const sinon = require('sinon');
+
+
+describe('App Tests', () => {
+  it('should contain App', () => {
+    const wrapper = shallow(<App />);
+    expect(wrapper.exists()).toBe(true);
+  });
+  it('has initial state watch set to false', () => {
```

Comment:
This test looks good right now! Just a note, you are probably aware,  if you decide to add functionality later to track for a given user which stocks they are already "watching", the initial state may not necessarily want to be false